rskj-core/src/test/java/co/rsk/peg/federation/PendingFederationTest.java

```diff
@@ -83,153 +74,169 @@ void membersImmutable() {
     }
 
     @Test
-    void isComplete() {
+    void isComplete_withAPendingFederationWithMoreMembersThanRequired_shouldBeTrue() {
         Assertions.assertTrue(pendingFederation.isComplete());
     }
 
     @Test
-    void isComplete_not() {
+    void isComplete_withAPendingFederationWithLessMembersThanRequired_shouldBeFalse() {
         PendingFederation otherPendingFederation = new PendingFederation(FederationTestUtils.getFederationMembersFromPks(200));
         Assertions.assertFalse(otherPendingFederation.isComplete());
     }
 
     @Test
-    void testEquals_basic() {
+    void equals_withTheSamePendingFederation_shouldBeTrue() {
```

Comment:
What about testing equals against each different type of PendingFed? Standard, Multisig, Erp, P2sh, Segwit.

Reply:
These were old tests that I just renamed and split. So far, there were no test cases for this